frontend/src/components/Landing/Landing.jsx

```diff
@@ -1,55 +1,157 @@
 import React, { useEffect } from 'react';
 import { observer } from 'mobx-react';
-import classnames from 'classnames';
 
-import { useStore } from 'stores/';
-import PsychologistFinder from './PsychologistFinder';
+import Slice from 'components/Slice/Slice';
+import Faq from 'components/Faq/Faq';
+import { useStore } from 'stores/index';
 import StudentProcess from './StudentProcess';
 import Statistics from './Statistics';
 
 import styles from './landing.cssmodule.scss';
 
-import Section from './Section';
+import Newsletter from './Newsletter';
 
 const Landing = () => {
   const { commonStore: { config } } = useStore();
-
   useEffect(() => {
     document.title = 'Santé Psy Étudiant';
   }, []);
 
   return (
-    <div className={classnames(styles.container, 'fr-container')} data-test-id="landingPageContainer">
-      <div className={styles.sectionAlt}>
-        <StudentProcess />
-      </div>
-      <div className={styles.section}>
-        <PsychologistFinder />
-      </div>
-      <div className={styles.sectionLight}>
-        <Section
-          title="Vous êtes psychologue ?"
-          description="Vous souhaitez participer au programme d&lsquo;accompagnement
-        psychologique des étudiants ? Merci !"
-          buttonAlignment="left"
-          buttonText="Commencer mon inscription"
-          buttonUrl={config.demarchesSimplifieesUrl}
-          buttonUrlExternal
-        />
-      </div>
-      <div className={styles.section}>
-        <Section
-          title="Vous avez une question ?"
-          description="Retrouvez les réponses aux questions les plus fréquemment posées."
-          buttonAlignment="right"
-          buttonText="Consulter la foire aux questions"
-          buttonUrl="/faq"
-          buttonSecondary
-        />
-      </div>
-      <div className={styles.sectionAlt}>
-        <Statistics />
-      </div>
+    <div className={styles.container} data-test-id="landingPageContainer">
+      <Slice
+        imageSrc="/images/landing.png"
+        title={(
+          <>
+            <div><b>Étudiants, étudiantes</b></div>
+            Besoin d’une oreille attentive ?
+          </>
+        )}
+        description={(
+          <>
+            Bénéficiez de
+            {' '}
+            <b>8 séances gratuites</b>
+            {' '}
+            avec une psychologue partenaire
```

Comment:
```suggestion
            avec un psychologue partenaire
```
L'erreur est sur la maquette aussi.
Peut être voir avec l'équipe si on mettrait pas "un ou une".